pkg/installation/install.go

```diff
@@ -113,18 +112,22 @@ func install(plugin, version, uri, bin string, p environment.Paths, fos []index.
 // Uninstall will uninstall a plugin.
 func Uninstall(p environment.Paths, name string) error {
 	if name == krewPluginName {
-		return errors.Errorf("removing krew is not allowed through krew. Please run:\n\t rm -r %s", p.BasePath())
+		glog.Errorf("Removing krew through krew is not supported.")
+		if !isWindows() { // assume POSIX-like
+			glog.Errorf("If you’d like to uninstall krew altogether, run:\n\trm -rf -- %q", p.BasePath())
+		}
+		return errors.New("self-uninstall not allowed")
 	}
 	glog.V(3).Infof("Finding installed version to delete")
-	version, installed, err := findInstalledPluginVersion(p.InstallPath(), p.BinPath(), name)
-	if err != nil {
-		return errors.Wrap(err, "can't uninstall plugin")
-	}
-	if !installed {
-		return ErrIsNotInstalled
+
+	if _, err := receipt.Load(p.PluginInstallReceiptPath(name)); err != nil {
```

Comment:
Again, a simpler `Lstat` would do.

Reply:
I'm inclined to keep this as is. If receipt cannot be parsed during uninstallation, that points to a problem and we should probably stop.

In the future we might introduce a field like `status.originIndex` (or like `status.installPath`) to determine the installation path. So we'll actually need to use the receipt, soon.

Reply:
Yeah, that makes sense. Then let's keep it as is.